starport/templates/module/create/oracle/obi/decode.go.plush

```diff
@@ -0,0 +1,190 @@
+package obi
```

Comment:
Can we import this as a pkg instead? If it's not hosted/maintained anywhere, we can push it to tendermint/spm.